docs/make.jl

```diff
@@ -1,6 +1,13 @@
-using OMOPCDMCohortCreator
 using Documenter
 
+try
+    using OMOPCDMCohortCreator
+catch
+    using Pkg
+    Pkg.add(url="github.com/JuliaHealth/OMOPCDMCohortCreator.jl", rev="main")
+    using OMOPCDMCohortCreator
+end
+
```

Comment:
I like where you are going with this but I am a bit curious - why do you think we need this? Did you run into issues with loading up the documentation locally on your side?

Reply:
It was tricky for me to start from scratch, as I aimed to just start Documenter and, of course, it was not willing to cooperate due to missing prerequisites.
If you think that's an overkill (and that could be one), we can leave it out.

Reply:
Yea - I will say let's leave this out. However, what were the missing prerequisites you ran into? Happy to help there.